### ISSUE TITLE
bug: `no-unused-props`'s option `checkImportedTypes` doesn't work

### ISSUE THREAD

**Issue by baseballyama** [sveltejs/eslint-plugin-svelte#1133]
Copied from https://github.com/sveltejs/eslint-plugin-svelte/issues/1028#issuecomment-2729172343
---

Then the checkImportedTypes option does not seem to work correctly. In the example below the svelte-component imports the TaskViewModel-type, but it still reports errors due to some of its attributes not being used

![Image](https://github.com/user-attachments/assets/e8a03557-8ea8-4111-a722-6b608f916823)

**Comment by baseballyama** [comment]
@shumadrid Could you please install the following package and confirm if it resolves your issue? I’d like to verify that the fix is correct before releasing it.

```shell
npm i https://pkg.pr.new/eslint-plugin-svelte@1135
```

**Comment by shumadrid** [comment]
Yes, it's fixed for me.

**Comment by MarcusCaspeco** [comment]
That one works for me as well, this is the case I have:

```svelte
<script lang="ts">
	import Component from "./component.svelte"

	interface Props {
		item: Item;
		// ...
	}
</script>

<Component {item} />
```

**Comment by baseballyama** [comment]
Nice! Thanks for reviewing! @shumadrid @MarcusCaspeco 
I’ll do a self-review and release it.
